src/ASM/LR/ASMu3Drecovery.C

```diff
@@ -157,25 +157,26 @@ bool ASMu3D::assembleL2matrices (SparseMatrix& A, StdVector& B,
       Go::BasisDerivs spl2;
       int ielp = group[t][e];
       const LR::Element* elm = proj->getElement(ielp);
-      int iel = lrspline->getElementContaining(elm->midpoint())+1;
+      int iel = lrspline->getElementContaining(elm->midpoint()) + 1;
+      int ielG = geo->getElementContaining(elm->midpoint()) + 1;
 
       if (continuous)
       {
         // Set up control point (nodal) coordinates for current element
         if (!this->getElementCoordinates(Xnod,iel)) {
           ok = false;
           continue;
-        } else if ((dV = 0.125*this->getParametricVolume(iel)) < 0.0) {
```

Comment:
OK, here goes the last `getParametricVolume()` usage.

Reply:
indeed.